### PR TITLE
Changes from new eoans gir-files

### PR DESCRIPTION
**Pull request by EPashkin**
@GuillaumeGomez, @sdroege 

These changes from updated eoans git-files (https://github.com/gtk-rs/gir-files/pull/54).

### CODE REVIEW

**Comment by sdroege** [comment]
Seems fine to me

**Comment by GuillaumeGomez** [comment]
The regen check doesn't seem happy?

**Comment by EPashkin** [comment]
There no version files, I add it when we merge gir-files part

**Comment by EPashkin** [comment]
Updated

File: atk-sys/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ shell-words = "0.1.0"
 tempfile = "3"
 
 [features]
+v2_35 = ["v2_34"]
```
Comment (sdroege) [inline atk-sys/Cargo.toml:18]: Why?

Comment (EPashkin) [reply atk-sys/Cargo.toml]: Good, catch, these focal changes

Comment (EPashkin) [reply atk-sys/Cargo.toml]: Updated

**Comment by sdroege** [comment]
What are all those failures in https://travis-ci.org/github/gtk-rs/sys/jobs/679749074 ?

**Comment by EPashkin** [comment]
Forgot to update submodules.
Should be fixed.

**Comment by sdroege** [comment]
Good for me when the CI is green :) Thanks!

**Comment by EPashkin** [comment]
And CI passed

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**